doc/stability_wasm_proposals.md

```diff
@@ -0,0 +1,74 @@
+# Wasm Proposals
+
+This document is intended to describe the current status of WebAssembly proposals and WASI proposals in WAMR.
+
+Only track proposals that are followed in the [WebAssembly proposals](https://github.com/WebAssembly/proposals) and [WASI proposals](https://github.com/WebAssembly/WASI/blob/main/Proposals.md).
+
+Normally, the document tracks proposals that are in phase 4. However, if a proposal in an earlier phase receives support, it will be added to the list below.
+
+The _status_ represents the configuration _product-mini/platforms/linux/CMakeLists.txt_. There may be minor differences between the top-level CMakeLists and platform-specific CMakeLists.
+
+Users can turn those features on or off by using compilation options. If a relevant compilation option is not available(`N/A`), it indicates that the feature is permanently enabled.
+
+## On-by-default Wasm Proposals
+
+| Proposal                              | >= Phase 4 | Compilation Option         |
+| ------------------------------------- | ---------- | -------------------------- |
+| Bulk memory operations                | Yes        | `WAMR_BUILD_BULK_MEMORY`   |
+| Extended Constant Expressions         | Yes        | N/A                        |
+| Fixed-width SIMD[^1]                  | Yes        | `WAMR_BUILD_SIMD`          |
+| Legacy Exception handling[^2]         | Yes        | `WAMR_BUILD_EXCE_HANDLING` |
```

Comment:
the "legacy" EH is its phase 3 version. it's misleading to say ">= Phase 4" "yes".

Reply:
agree, it's a bit confusing. Also, given the "legacy" EH proposal is **legacy** and never actually was standardized, should we make it off by default to discourage people from using it?

Reply:
Please share your thoughts on the most recent version. 🤝